.github/workflows/main.yml

```diff
@@ -0,0 +1,12 @@
+name: Label PRs
+
+on:
+  - pull_request
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+	steps:
```

Comment:
perhaps these are tabs and it wants spaces?

Reply:
Well that's the last time I do a copy paste from .readme. Think that was it